_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+
+Although the part about equipments is relatively straight-forward, it is key to properly negotiate and identify all the contractual elements associated with hardware purchases.
+
+## RFI: Request For Information
+
+A RFI is usually used to consult vendors on the feasibility of a project and get an estimate of the associated costs. A RFI could eventually be sent to a large number of vendors and used as a mean of shortlisting vendors in preparation for a RFP or RFQ.
+For example, sending the RFI to 15 vendors with the objective of keeping to invite the 3 best vendors for a RFP or RFQ phase.
+
+> I would like to build a Compute and Storage cloud capable of storing 6 PB of data and running 1000 Virtual Machines in parallel.
+
+## RFP: Request For Proposal
+
+A RFP is usually used to obtain detailed pricing and conditions from vendors on complex projects. Exact specifications of the components are potentially unknown (or partially known) by the buyer who relies on the Vendor's expertise to identify the most relevant solution.
+
+> I would like to build an OpenStack Compute and Storage cluster capable of storing 6 PB or data and running 1000 Virtual Machines in parallel. The infrastructure should be interconnected with high-performance networking providing at least 10 Gbps connectivity to the servers and 100 Gbps inter-rack connectivity. Storage should rely on 6 to 8 TB SAS drives. Compute servers should be composed of Intel CPUs with 10-14 cores and contain at least 256GB or RAM per server.
+
+## RFQ: Request For Quotation
+
+A RFQ is usually used to obtain detailed pricing from vendors based on specifications sent by the buyer. This methods relies on the buyer clearly understanding his needs, requirements and components specifications.
+
+> Please provide a quote for IBM servers with Intel Xeon E5-2670 CPUs, 256GB of DDR4-2133 ECC RAM, 6x 2TB 2.5" SAS drives (12GB/s, 7.2K RPM, 128MB Cache)...
+
+## RFSQ: Request For Supplier Qualification
+
+A RFSQ is usually used to pre-select vendors and establish common contractual agreement for multiple purchases made under this agreement. By establishing contractual details upfront, it facilitates subsequent quotation phases, which are all based on the same conditions (support, warranty period, lead times, ...).
+
+The initial stage of a RFSQ has similarities with a RFP in the sense that it contains contractual details and, in our situation, relied on a "simulated" proposal to evaluate vendors. Following the RFSQ, subsequent procurement is executed through multiple RFQ phases.
```

Comment:
The initial stage of a RFSQ has similarities with a RFP in the sense that it contains contractual details and, in our situation, it relied on a "simulated" proposal to evaluate vendors. Following the RFSQ, subsequent procurement is executed through multiple RFQ phases